index.json

```diff
@@ -73,6 +73,7 @@
     "@elizaos-plugins/plugin-letzai": "github:elizaos-plugins/plugin-letzai",
     "@elizaos-plugins/plugin-lightlink": "github:lightlink-network/plugin-lightlink",
     "@elizaos-plugins/plugin-massa": "github:elizaos-plugins/plugin-massa",
+    "@elizaos-plugins/plugin-mcp": "github:fleek-platform/plugin-mcp",
```

Comment:
hey @gabrielmpinto good work 

change url to : github:fleek-platform/eliza-plugin-mcp

ref: https://github.com/fleek-platform/eliza-plugin-mcp

Reply:
Damn, I had one job 😄 thanks, fixed it

Reply:
perfect!